pkg/bootloader/grub.go

```diff
@@ -24,6 +24,8 @@ import (
 	"path/filepath"
 	"strings"
 
+	"golang.org/x/exp/slices"
```

Comment:
Wasn't it enough with "slices"? I think Reverse is already part of the [standard library](https://pkg.go.dev/slices#Reverse)